roles/config-docker-compose/README.md

```diff
@@ -0,0 +1,56 @@
+config-docker-compose
+=====================
+
+The config-docker-compose role is associated with installing the docker compose packages.
+
+Requirements
+------------
+
+N/A
+
+
+Role Variables
+--------------
+
+Defaults file has example of variables that can be used.
+
+| Variable | Description | Required | Defaults |
+|:--------:|:-----------:|:--------:|:--------:|
+|**docker_compose_packages**|  Packages required for docker compose. | yes | in defaults file |
+|**docker_install**| Should docker be installed | true | false |
```

Comment:
`true` should probably be `yes` as this is for humans to read .:-) - same comment for one line below

Reply:
Also, same as above, should probably say "no" for `Required` as there is a default value.

Reply:
Changed as requested.